src/base/system_parameter.c

```diff
@@ -737,6 +737,8 @@ static const char sysprm_ha_conf_file_name[] = "cubrid_ha.conf";
 #define PRM_NAME_DEDUPLICATE_KEY_LEVEL     "deduplicate_key_level"
 #define PRM_NAME_PRINT_INDEX_DETAIL        "print_index_detail"
 
+#define PRM_NAME_MAX_SUBQUERY_CACHE_SIZE    "subquery_cache_size"
```

Comment:
Please use the same name as the macro variable name and parameter name.

Reply:
I'll name it "max_subquery_cache_size"